builds/e2e/proxy/configure_runner.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euo pipefail
+
+proxy="http://${1}:3128"
+
+export http_proxy=$proxy
+export https_proxy=$proxy
+
+echo 'Installing PowerShell Core and .NET Core 3.1'
+
+apt-get update
+apt-get install -y curl git wget apt-transport-https
+wget -q 'https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb'
+dpkg -i packages-microsoft-prod.deb
+apt-get update
+add-apt-repository universe
+apt-get install -y powershell dotnet-sdk-3.1
```

Comment:
I thought these (Line 12-18) are one-time prerequisite for a VM to run E2E test. 
I don't think we need to install that every time the test is run?

Reply:
This script is part of the ARM (Azure Resource Manager) automated deployment for the proxy environment. This script is only run when the VMs are created. It is not part of the end-to-end tests. Everything in the `proxy/` folder is part of the deployment, not the tests.